modules/nf-core/mindagap/mindagap/meta.yml

```diff
@@ -0,0 +1,49 @@
+name: "mindagap_mindagap"
+description: Takes a single panorama image and fills the empty grid lines with neighbour-weighted values.
+keywords:
+  - imaging
+  - resolve_bioscience
+  - spatial_transcriptomics
+tools:
+  - "mindagap":
+      description: "Mindagap is a collection of tools to process multiplexed FISH data, such as produced by Resolve Biosciences Molecular Cartography."
+      homepage: "https://github.com/ViriatoII/MindaGap"
+      documentation: "https://github.com/ViriatoII/MindaGap/blob/main/README.md"
+      tool_dev_url: "https://github.com/ViriatoII/MindaGap"
+      doi: ""
+      licence: "BSD-3-Clause license"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - tiff:
+      type: file
+      description: A tiff file containing gridlines as produced by Molecular Cartography imaging.
+      pattern: "*.{tiff}"
+
+  # args = optional arguments
+  # args2 = positional arguments (required) s (box size for gaussian kernel) and r (number of rounds to apply gaussianBlur)
```

Comment:
according to the new guidelines all mandatory non-file arguments also need to be given as input values to the module. Can you add these values as inputs?

Reply:
I made the changes to add s and r as mandatory input parameter integers!